tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -371,3 +411,220 @@ ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_i
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+	zassert_equal_ptr(NULL,
+			  mock_cap_commander_broadcast_reception_start_cb_fake.arg0_history[0]);
+	zassert_equal(0, mock_cap_commander_broadcast_reception_start_cb_fake.arg1_history[0]);
+
+	/* We test with one subgroup, instead of CONFIG_BT_BAP_BASS_MAX_SUBGROUPS subgroups */
+	for (size_t i = 0U; i < CONFIG_BT_MAX_CONN; i++) {
+		fixture->stop_param.param[i].num_subgroups = 1;
+		fixture->stop_param.param[i].src_id = src_id[i];
+	}
+
+	err = bt_cap_commander_broadcast_reception_stop(&fixture->stop_param);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_stop", 1,
+			   mock_cap_commander_broadcast_reception_stop_cb_fake.call_count);
+	zassert_equal_ptr(NULL,
+			  mock_cap_commander_broadcast_reception_stop_cb_fake.arg0_history[0]);
+	zassert_equal(0, mock_cap_commander_broadcast_reception_stop_cb_fake.arg1_history[0]);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_stop_double)
+{
+	int err;
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 1,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+	zassert_equal_ptr(NULL,
+			  mock_cap_commander_broadcast_reception_start_cb_fake.arg0_history[0]);
+	zassert_equal(0, mock_cap_commander_broadcast_reception_start_cb_fake.arg1_history[0]);
+
+	for (size_t i = 0U; i < CONFIG_BT_MAX_CONN; i++) {
+		printk("Source ID %d: %d %d\n", i, fixture->stop_param.param[i].src_id, src_id[i]);
+
+		fixture->stop_param.param[i].src_id = src_id[i];
+	}
+
+	err = bt_cap_commander_broadcast_reception_stop(&fixture->stop_param);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	err = bt_cap_commander_broadcast_reception_stop(&fixture->stop_param);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_stop", 1,
+			   mock_cap_commander_broadcast_reception_stop_cb_fake.call_count);
+
+	/*
+	 * Since the 2nd reception_stop procedure never completed initiating a new CAP procedure
+	 * should result in an error -EBUSY
+	 */
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EBUSY, err, "Unexpected return value %d", err);
+
+	/*
+	 * We need to cleanup since the CAP commander remembers state, otherwise the next test will
+	 * fail.
+	 * Since this cleanup is not normally necessary we do it here and not in
+	 * cap_commander_test_broadcast_reception_after
+	 */
+	err = bt_cap_commander_cancel();
+	zassert_equal(0, err, "Unexpected return value: %d", err);
+}
```

Comment:
Consider always calling `cancel` in the cleanup function of the test suite. If any tests fails for any reason, then it will be left in a `busy` state. 

We should always attempt to clean up everything after any test so that if Test A fails it does not affect Test B